docs/source/en/installation.mdx

```diff
@@ -139,15 +139,15 @@ conda install -c huggingface transformers
 
 ## Cache setup
 
-Pretrained models are downloaded and locally cached at: `~/.cache/huggingface/transformers/`. This is the default directory given by the shell environment variable `TRANSFORMERS_CACHE`. On Windows, the default directory is given by `C:\Users\username\.cache\huggingface\transformers`. You can change the shell environment variables shown below - in order of priority - to specify a different cache directory:
+Pretrained models are downloaded and locally cached at: `~/.cache/huggingface/`. This is the default directory given by the shell environment variable `HUGGINGFACE_CACHE`. On Windows, the default directory is given by `C:\Users\username\.cache\huggingface`. You can change the shell environment variables shown below - in order of priority - to specify a different cache directory:
```

Comment:
```suggestion
Pretrained models are downloaded and locally cached at: `~/.cache/huggingface/hub/`. This is the default directory given by the shell environment variable `HUGGINGFACE_HUB_CACHE`. On Windows, the default directory is given by `C:\Users\username\.cache\huggingface\hub`. You can change the shell environment variables shown below - in order of priority - to specify a different cache directory:
```

For consistency with, and to use the same cache as, `huggingface_hub`?

Reply:
I thought we wanted the root, and have other libraries migrate to it too?